src/utils/user.ts

```diff
@@ -0,0 +1,48 @@
+import cheerio, { CheerioAPI } from "cheerio";
+
+import axios from "axios";
+
+interface UserProfile extends CheerioAPI {
+  kind: "UserProfile";
+}
+
+export interface User {
+ name: string;
+ pseuds: string;
```

Comment:
You probably want this to be a `string[]`

Reply:
Okay! a) All of them? b) Why?

Reply:
a) no, just pseuds
b) cause you can have more than one, right?

You'll have to decide whether the original username will appear in pseuds. IDK which one is consistent with how ao3 does it, but write a note to document it.

Reply:
Original appearing is consistent with ao3, will do